microsoft-365/security/defender/api-advanced-hunting.md

```diff
@@ -48,11 +48,11 @@ The following conditions relate to all queries.
 6. A `429` HTTP response code indicates that you've reached a quota, either by number of requests sent, or by allotted running time. Read the response body to understand the limit you have reached. 
 
 > [!NOTE]
-> All quotas listed above (for example 15 calls per min) are per tenant size. These quotas are the minimum.
+> All quotas listed above (for example 15 calls per min) are tenant wide. These quotas are the minimum.
 
 ## Permissions
 
-One of the following permissions is required to call the advanced hunting API. To learn more, including how to choose permissions, see [Access the Microsoft 365 Defender Protection APIs](api-access.md)
+One of the following permissions is required to call the Advanced Hunting API. To learn more, including how to choose permissions, see [Access the Microsoft 365 Defender Protection APIs](api-access.md)
```

Comment:
```suggestion
One of the following permissions is required to call the advanced hunting API. To learn more, including how to choose permissions, see [Access the Microsoft 365 Defender Protection APIs](api-access.md).
```

Reply:
Hey @JohanFreelancer9 ,
Just wondering, why Advanced Hunting should be in lower case? Isn't it a personal name of this API?

Reply:
Hey @YuriySamorodov, thank you for the question. I know capitalization of this term might be somewhat inconsistent throughout the documentation, but the latest guidance is that this is a descriptive name, like many other capabilities, not a product name, and should not be capitalized. Thanks again.

Reply:
Thanks @JohanFreelancer9 , so in other words Advanced Hunting is not the specific API name?

Reply:
@YuriySamorodov, that is correct, it is a descriptive name.

Reply:
Make sense now, thank you @JohanFreelancer9